arcade/examples/tetris.py

```diff
@@ -11,6 +11,8 @@
 import arcade
 import random
 import PIL
+import pyglet
+import time
```

Comment:
For examples that demonstrate a certain concept, like how-to-do-Tetris I'd like to keep things narrow. Basic concept, as easy as possible to understand.

This should demo how to do a basic Tetris. If we want to center the window on the screen, adjust timings, etc. that should be a separate demo. (As these imports are related to.)

A full bells-and-whistles Tetris should be done as a different example. Probably in a different repository.

A step-by-step tutorial would be another, even more useful, option.